eng/pipelines/templates/scripts/create-tags-and-git-release.yml

```diff
@@ -0,0 +1,11 @@
+steps:
```

Comment:
Can you list the parameters out here so it is easy for folks that want to use this template to know what they need to specify.